### ISSUE TITLE
Scraping error

### ISSUE THREAD

**Issue by Gingerdilligaf** [ResetCollection/support#61]
Problems with scraping systems. Sticking on please wait when adding new system

**Comment by ReCoJav** [comment]
Hello!

Really sorry you're experiencing this issue! I definitely want to get it fixed!

Is this still an issue for you? If so, can I get a bit more info, please?

And some questions:
Is this happening for all systems you've tried adding?
What system is this happening on?
What device are you using?

Thanks!

**Comment by Gingerdilligaf** [comment]
HiIt's happening on both shield tv & samsung tablet. I've added between 25 to 30 systems without a problem & now it struggles to add more. It does eventually scrape but sticks on the please wait screen for up to 20mins before scrapingCheersSent from my Samsung Galaxy smartphone.
-------- Original message --------From: ReCoJav <notifications@github.com> Date: 14/04/2020  09:19  (GMT+00:00) To: ResetCollection/support <support@noreply.github.com> Cc: Gingerdilligaf <dylandouglass@hotmail.co.uk>, Author <author@noreply.github.com> Subject: Re: [ResetCollection/support] Scraping error (#61) 
Hello!
Really sorry you're experiencing this issue! I definitely want to get it fixed!
Is this still an issue for you? If so, can I get a bit more info, please?
And some questions:
Is this happening for all systems you've tried adding?
What system is this happening on?
What device are you using?
Thanks!

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/ResetCollection/support/issues/61#issuecomment-613296300",
"url": "https://github.com/ResetCollection/support/issues/61#issuecomment-613296300",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by ReCoJav** [comment]
@Gingerdilligaf 

Alrighty.... looks like I'll try to recreate this by adding a lot of systems! I've never added that many systems. But I know there are some other power users out there adding their whole collection on a Shield TV.... but I should definitely test this out. I have a Shield TV and Samsung tablet, so when I have some time I'll do a huge test!

Oh, can I ask... how many games do you think you have per system? Are you doing full rom sets? Thanks!

**Comment by Gingerdilligaf** [comment]
Yeah the majority are full romsets as I used to have a hyperspin setup but found it over complicated. On shield I think the problem started after I'd added a mame collection. On tablet I think it was a large n64 set.Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: ReCoJav <notifications@github.com> Date: 16/04/2020  01:47  (GMT+00:00) To: ResetCollection/support <support@noreply.github.com> Cc: Gingerdilligaf <dylandouglass@hotmail.co.uk>, Mention <mention@noreply.github.com> Subject: Re: [ResetCollection/support] Scraping error (#61) 
@Gingerdilligaf
Alrighty.... looks like I'll try to recreate this by adding a lot of systems! I've never added that many systems. But I know there are some other power users out there adding their whole collection on a Shield TV.... but I should definitely test this out. I have a Shield TV and Samsung tablet, so when I have some time I'll do a huge test!
Oh, can I ask... how many games do you think you have per system? Are you doing full rom sets? Thanks!

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/ResetCollection/support/issues/61#issuecomment-614350625",
"url": "https://github.com/ResetCollection/support/issues/61#issuecomment-614350625",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Davemk5955** [comment]
Also having the same issue although does eventually start to scrape. (using Nvidia Shield)

**Comment by Gingerdilligaf** [comment]
Yeah it does eventually. Frustrating because I actually really like the app & I always happy to help. At the start I thought it maybe a server error with thegamesdb but I tried add a system on a samsung phone & it was flawlessSent from my Samsung Galaxy smartphone.
-------- Original message --------From: Davemk5955 <notifications@github.com> Date: 16/04/2020  22:29  (GMT+00:00) To: ResetCollection/support <support@noreply.github.com> Cc: Gingerdilligaf <dylandouglass@hotmail.co.uk>, Mention <mention@noreply.github.com> Subject: Re: [ResetCollection/support] Scraping error (#61) 
Also having the same issue although does eventually start to scrape. (using Nvidia Shield)

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/ResetCollection/support/issues/61#issuecomment-614906226",
"url": "https://github.com/ResetCollection/support/issues/61#issuecomment-614906226",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Gingerdilligaf** [comment]
Also if u back out of the app when its stuck on please wait...when u reopen the app its scraped the youtube videos? Might be the youtube links?Dont know if it's possible but if that's causing the error maybe a link to emumovies?Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Davemk5955 <notifications@github.com> Date: 16/04/2020  22:29  (GMT+00:00) To: ResetCollection/support <support@noreply.github.com> Cc: Gingerdilligaf <dylandouglass@hotmail.co.uk>, Mention <mention@noreply.github.com> Subject: Re: [ResetCollection/support] Scraping error (#61) 
Also having the same issue although does eventually start to scrape. (using Nvidia Shield)

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/ResetCollection/support/issues/61#issuecomment-614906226",
"url": "https://github.com/ResetCollection/support/issues/61#issuecomment-614906226",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Davemk5955** [comment]
> Also if u back out of the app when its stuck on please wait...when u reopen the app its scraped the youtube videos? Might be the youtube links?Dont know if it's possible but if that's causing the error maybe a link to emumovies?Sent from my Samsung Galaxy smartphone. -------- Original message --------From: Davemk5955 <notifications@github.com> Date: 16/04/2020 22:29 (GMT+00:00) To: ResetCollection/support <support@noreply.github.com> Cc: Gingerdilligaf <dylandouglass@hotmail.co.uk>, Mention <mention@noreply.github.com> Subject: Re: [ResetCollection/support] Scraping error (#61) Also having the same issue although does eventually start to scrape. (using Nvidia Shield) —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe. [ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "[#61 (comment)](https://github.com/ResetCollection/support/issues/61#issuecomment-614906226)", "url": "[#61 (comment)](https://github.com/ResetCollection/support/issues/61#issuecomment-614906226)", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

Yeah. I did notice backing out and coming back in it seems to have then scrapped. odd tho as when I first used it yesterday was very fast. 

(I do have average internet tho understandable if it is something to do with youtube.)

Wondering if there will be an option in the future to add own videos rather than the youtube option.

**Comment by Gingerdilligaf** [comment]
This app was recommended to me on a launchbox forum by Jason Carr. I know he gave up on launchbox for android but he seems really helpful. We all wanting a great frontend on android & to be fair yours is best so far. I dont know if he could maybe help with any issues that crop up? I follow eta prime on youtube & I want to see this app get the promotion it deserves Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Davemk5955 <notifications@github.com> Date: 16/04/2020  23:21  (GMT+00:00) To: ResetCollection/support <support@noreply.github.com> Cc: Gingerdilligaf <dylandouglass@hotmail.co.uk>, Mention <mention@noreply.github.com> Subject: Re: [ResetCollection/support] Scraping error (#61) 



Also if u back out of the app when its stuck on please wait...when u reopen the app its scraped the youtube videos? Might be the youtube links?Dont know if it's possible but if that's causing the error maybe a link to emumovies?Sent from my Samsung Galaxy smartphone. -------- Original message --------From: Davemk5955 notifications@github.com Date: 16/04/2020 22:29 (GMT+00:00) To: ResetCollection/support support@noreply.github.com Cc: Gingerdilligaf dylandouglass@hotmail.co.uk, Mention mention@noreply.github.com Subject: Re: [ResetCollection/support] Scraping error (#61) Also having the same issue although does eventually start to scrape. (using Nvidia Shield) —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe. [ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "#61 (comment)", "url": "#61 (comment)", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]



Yeah. I did notice backing out and coming back in it seems to have then scrapped. odd tho as when I first used it yesterday was very fast.

(I do have average internet tho understandable if it is something to do with youtube.)

Wondering if there will be an option in the future to add own videos rather than the youtube option.



—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

[

{

"@context": "http://schema.org",

"@type": "EmailMessage",

"potentialAction": {

"@type": "ViewAction",

"target": "https://github.com/ResetCollection/support/issues/61#issuecomment-614927068",

"url": "https://github.com/ResetCollection/support/issues/61#issuecomment-614927068",

"name": "View Issue"

},

"description": "View this Issue on GitHub",

"publisher": {

"@type": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]

**Comment by Davemk5955** [comment]
> This app was recommended to me on a launchbox forum by Jason Carr. I know he gave up on launchbox for android but he seems really helpful. We all wanting a great frontend on android & to be fair yours is best so far. I dont know if he could maybe help with any issues that crop up? I follow eta prime on youtube & I want to see this app get the promotion it deserves Sent from my Samsung Galaxy smartphone. -------- Original message --------From: Davemk5955 <notifications@github.com> Date: 16/04/2020 23:21 (GMT+00:00) To: ResetCollection/support <support@noreply.github.com> Cc: Gingerdilligaf <dylandouglass@hotmail.co.uk>, Mention <mention@noreply.github.com> Subject: Re: [ResetCollection/support] Scraping error (#61) Also if u back out of the app when its stuck on please wait...when u reopen the app its scraped the youtube videos? Might be the youtube links?Dont know if it's possible but if that's causing the error maybe a link to emumovies?Sent from my Samsung Galaxy smartphone. -------- Original message --------From: Davemk5955 notifications@github.com Date: 16/04/2020 22:29 (GMT+00:00) To: ResetCollection/support support@noreply.github.com Cc: Gingerdilligaf dylandouglass@hotmail.co.uk, Mention mention@noreply.github.com Subject: Re: [ResetCollection/support] Scraping error (#61) Also having the same issue although does eventually start to scrape. (using Nvidia Shield) —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe. [ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "#61 (comment)", "url": "#61 (comment)", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ] Yeah. I did notice backing out and coming back in it seems to have then scrapped. odd tho as when I first used it yesterday was very fast. (I do have average internet tho understandable if it is something to do with youtube.) Wondering if there will be an option in the future to add own videos rather than the youtube option. —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe. [ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "[#61 (comment)](https://github.com/ResetCollection/support/issues/61#issuecomment-614927068)", "url": "[#61 (comment)](https://github.com/ResetCollection/support/issues/61#issuecomment-614927068)", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

Yeah ditto. same here, it was a shame that Launchbox/Android was stopped. Had high hopes for a bigbox mode. Would have loved to have seen that. Still a huge fan of Launchbox on Windows, (great community)  Reset Collection however is a great frontend for Android.

**Comment by Gingerdilligaf** [comment]
Great app. People are giving up on android rather than getting together & sharing knowledge. Big up to reset collection & arc browser & hoping they create the perfect android app together :-)Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Davemk5955 <notifications@github.com> Date: 16/04/2020  23:40  (GMT+00:00) To: ResetCollection/support <support@noreply.github.com> Cc: Gingerdilligaf <dylandouglass@hotmail.co.uk>, Mention <mention@noreply.github.com> Subject: Re: [ResetCollection/support] Scraping error (#61) 



This app was recommended to me on a launchbox forum by Jason Carr. I know he gave up on launchbox for android but he seems really helpful. We all wanting a great frontend on android & to be fair yours is best so far. I dont know if he could maybe help with any issues that crop up? I follow eta prime on youtube & I want to see this app get the promotion it deserves Sent from my Samsung Galaxy smartphone. -------- Original message --------From: Davemk5955 notifications@github.com Date: 16/04/2020 23:21 (GMT+00:00) To: ResetCollection/support support@noreply.github.com Cc: Gingerdilligaf dylandouglass@hotmail.co.uk, Mention mention@noreply.github.com Subject: Re: [ResetCollection/support] Scraping error (#61) Also if u back out of the app when its stuck on please wait...when u reopen the app its scraped the youtube videos? Might be the youtube links?Dont know if it's possible but if that's causing the error maybe a link to emumovies?Sent from my Samsung Galaxy smartphone. -------- Original message --------From: Davemk5955 notifications@github.com Date: 16/04/2020 22:29 (GMT+00:00) To: ResetCollection/support support@noreply.github.com Cc: Gingerdilligaf dylandouglass@hotmail.co.uk, Mention mention@noreply.github.com Subject: Re: [ResetCollection/support] Scraping error (#61) Also having the same issue although does eventually start to scrape. (using Nvidia Shield) —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe. [ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "#61 (comment)", "url": "#61 (comment)", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ] Yeah. I did notice backing out and coming back in it seems to have then scrapped. odd tho as when I first used it yesterday was very fast. (I do have average internet tho understandable if it is something to do with youtube.) Wondering if there will be an option in the future to add own videos rather than the youtube option. —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe. [ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "#61 (comment)", "url": "#61 (comment)", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]



Yeah ditto. same here, it was a shame that Launchbox/Android was stopped. Had high hopes for a bigbox mode. Would have loved to have seen that. Still a huge fan of Launchbox on Windows, (great community)  Reset Collection however is a great frontend for Android.



—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

[

{

"@context": "http://schema.org",

"@type": "EmailMessage",

"potentialAction": {

"@type": "ViewAction",

"target": "https://github.com/ResetCollection/support/issues/61#issuecomment-614933163",

"url": "https://github.com/ResetCollection/support/issues/61#issuecomment-614933163",

"name": "View Issue"

},

"description": "View this Issue on GitHub",

"publisher": {

"@type": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]

**Comment by ReCoJav** [comment]
> > Also if u back out of the app when its stuck on please wait...when u reopen the app its scraped the youtube videos? Might be the youtube links?Dont know if it's possible but if that's causing the error maybe a link to emumovies?Sent from my Samsung Galaxy smartphone. -------- Original message --------From: Davemk5955 [notifications@github.com](mailto:notifications@github.com) Date: 16/04/2020 22:29 (GMT+00:00) To: ResetCollection/support [support@noreply.github.com](mailto:support@noreply.github.com) Cc: Gingerdilligaf [dylandouglass@hotmail.co.uk](mailto:dylandouglass@hotmail.co.uk), Mention [mention@noreply.github.com](mailto:mention@noreply.github.com) Subject: Re: [ResetCollection/support] Scraping error (#61) Also having the same issue although does eventually start to scrape. (using Nvidia Shield) —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe. [ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "[#61 (comment)](https://github.com/ResetCollection/support/issues/61#issuecomment-614906226)", "url": "[#61 (comment)](https://github.com/ResetCollection/support/issues/61#issuecomment-614906226)", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
> 
> ****_Yeah. I did notice backing out and coming back in it seems to have then scrapped. odd tho as when I first used it yesterday was very fast.
> 
> (I do have average internet tho understandable if it is something to do with youtube.)
> 
> Wondering if there will be an option in the future to add own videos rather than the youtube option._****

Yup, I will one day give the option to add your own videos that are located on your device!

For now, you can add/update which YouTube video plays:
https://www.youtube.com/watch?v=mIgttRQLNkk&t=653s
(5:59 - Add or change video background)

And I'll try to respond and to the rest of this thread, and also work out the kink with the slow scraping! Sorry y'all are having this issue!!! RESET is a pretty young app, and as more users use it, I'm getting more feature requests and more people finding issues! I promise I'll be fixing, updating, and adding features for years to come! So RESET 1 year and 2 years from now is going to be much different, better, and with more features! And you'll never have to pay more for anything (and never ever will there be an ad or donate button!)

Thanks all! and love that y'all are talking about this and other frontends and retro gaming related stuff!

**Comment by Davemk5955** [comment]
> Yup, I will one day give the option to add your own videos that are located on your device!
> 
> For now, you can add/update which YouTube video plays:
> https://www.youtube.com/watch?v=mIgttRQLNkk&t=653s
> (5:59 - Add or change video background)
> 
> And I'll try to respond and to the rest of this thread, and also work out the kink with the slow scraping! Sorry y'all are having this issue!!! RESET is a pretty young app, and as more users use it, I'm getting more feature requests and more people finding issues! I promise I'll be fixing, updating, and adding features for years to come! So RESET 1 year and 2 years from now is going to be much different, better, and with more features! And you'll never have to pay more for anything (and never ever will there be an ad or donate button!)
> 
> Thanks all! and love that y'all are talking about this and other frontends and retro gaming related stuff!

Awesome! cant wait, thank you for the commitment, you certainly have my support and praise, Reset is already a great frontend and look forward to new features and updates, Nothing but love for all retro gaming and all the work that everyone puts into it.

**Comment by Davemk5955** [comment]
> For now, you can add/update which YouTube video plays:
> https://www.youtube.com/watch?v=mIgttRQLNkk&t=653s
> (5:59 - Add or change video background)

Have already changed some to custom cinematic themes which I uploaded on my youtube page, hence why I asked about adding own videos in the future which will certainly make this frontend very awesome indeed. :)
